docs/guides/hooks-list.rst

```diff
@@ -186,6 +182,30 @@ Action hooks
 **forward, <reason>**
 	Filter the URL to forward a user to when ``forward($url, $reason)`` is called.
 
+.. _guides/hooks-list#ajax:
+
+Ajax
```

Comment:
I think we need to document client-side hooks in a new section. 

> This is *not* triggered by the ``elgg/Ajax`` module.
IMHO, this shouldn't be necessary, if we document hooks separately.